docs/kfdrc_lancet_sub_wf.md

```diff
@@ -99,4 +94,4 @@ outputs:
 
  - `lancet_prepass_vcf`: VCF with SNV, MNV, and INDEL variant calls. Contains all soft `FILTER` values generated by variant caller, Use this file if you believe important variants are being left out when using the algorithm's `PASS` filter
  - `lancet_protected_outputs`: Array of files containing MAF format of PASS hits, `PASS` VCF with annotation pipeline soft `FILTER`-added values, and VCF index
- - `lancet_public_outputs`: Same as above, except MAF and VCF have had entries with soft `FILTER` values removed
```

Comment:
Weird that this was missed!

Reply:
Yeah I don't know what github thinks is being changed here. Maybe there was trailing whitespace?